Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -893,3 +911,15 @@ class ASHeaderView: UICollectionReusableView {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+open class PinnedSite: Site {
+    let isPinnedSite: Bool
```

Comment:
Can probably just do `let isPinnedSite = true` to make this more declarative since the main reason for this class is to distinguish between it and a top site.